core/test/routing.spec.js

```diff
@@ -131,6 +135,43 @@ describe('Routing', () => {
         component.get().hideNav,
         window.Luigi.config.settings.hideNavigation
       );
+      assert.equal(component.get().showLoadingIndicator, true);
```

Comment:
If possible, please try not to add this additional assert to this test, and for testing the loading indicator logic create two tests as small as possible

Reply:
I'll leave it, since loading indicator is standard now and enabled by default